### ISSUE TITLE
Changing Google Account Associated for Authentication

### ISSUE THREAD

**Issue by DrChrisLevy** [rstats-db/bigrquery#136]
When I first tried bigrquery I must of authenticated with a personal gmail/google account:

The first time you use bigrquery in a session, it will ask you to authorize bigrquery in the browser. This gives bigrquery the credentials to access data on your behalf. By default, bigrquery picks up httr's policy of caching per-working-directory credentials in .httr-oauth.

When I try to run my simple query I get the error:

Error: Access Denied: Project project_name: The user my_personal_gmail does not have bigquery.jobs.create permission in project project_name.

This is because my_personal_gmail does not have access to the project on bigquery. 
I need to change the my_personal_gmail to the google account that has access. 

Im assuming I need to do this within R. 

Any help would be awesome.

Thanks

**Comment by BrianWeinstein** [comment]
If you delete the `.httr-oauth` file (it should be in your working directory), you'll be prompted to re-authorize the next time you try to run a query.

You _should_ also be able to run `bigrquery::reset_access_cred()` during your R session to remove the environment variable that stores your credentials, but for some reason it's not working for me at the moment.

**Comment by DrChrisLevy** [comment]
Hey Brian

Thanks for the quick response.
Your suggestion above about removing the file .httr-oauth  worked and was very helpful.

The file was hidden at first in the mac finder. After revealing the file, deleting it, and restarting Rstudio, it all seems to be working.

**Comment by D6Analytics** [comment]
> If you delete the `.httr-oauth` file (it should be in your working directory), you'll be prompted to re-authorize the next time you try to run a query.
> 
> You _should_ also be able to run `bigrquery::reset_access_cred()` during your R session to remove the environment variable that stores your credentials, but for some reason it's not working for me at the moment.

@BrianWeinstein I've got a similar issue, but not related to bigrquery, I just found this thread.

I've got the same issue, where I authenticated to the wrong Google account. I tried deleting the auth file and re-running, but it never asked me to re-authenticate. Any thoughts?

I connect to Google Analytics in R via:

setwd("C:/D6/GoogAnalytics")
client.id <- "0000000-xxxxxxxx.apps.googleusercontent.com"
client.secret <- "xx000xxx0000xx"
google <- oauth_endpoints("google")
google_app <- oauth_app("googanalytics",client.id,client.secret)
google_scope <- "https://www.googleapis.com/auth/analytics"
oauth_token <- oauth2.0_token(google,google_app,google_scope,cache=T)
oauth_token$refresh()
